entity-framework/core/managing-schemas/migrations/managing.md

```diff
@@ -0,0 +1,199 @@
+
+```c#
+migrationBuilder.RenameColumn(
+    name: "Name",
+    table: "Customers",
+    newName: "FullName");
+```
+
+> [!TIP]
+> The migration scaffolding process warns when an operation might result in data loss (like dropping a column). If you see that warning, be especially sure to review the migrations code for accuracy.
+
+### Adding raw SQL
+
+While renaming a column can be achieved via a built-in API, in many cases that is not possible. For example, we may want to replace existing `FirstName` and `LastColumn` properties with a single, new `FullName` property. The migration generated by EF Core will be the following:
+
+``` csharp
+migrationBuilder.DropColumn(
+    name: "FirstName",
+    table: "Customer");
+
+migrationBuilder.DropColumn(
+    name: "LastName",
+    table: "Customer");
+
+migrationBuilder.AddColumn<string>(
+    name: "Name",
+    table: "Customer",
+    nullable: true);
+```
+
+As before, this would cause unwanted data loss. To transfer the data from the old columns, we rearrange the migrations and introduce a raw SQL operation as follows:
+
+``` csharp
+migrationBuilder.AddColumn<string>(
+    name: "Name",
+    table: "Customer",
+    nullable: true);
+
+migrationBuilder.Sql(
+@"
+    UPDATE Customer
+    SET Name = FirstName + ' ' + LastName;
+");
+
+migrationBuilder.DropColumn(
+    name: "FirstName",
+    table: "Customer");
+
+migrationBuilder.DropColumn(
+    name: "LastName",
+    table: "Customer");
+```
+
+### Arbitrary changes via raw SQL
+
+Raw SQL can also be used to manage database objects that EF Core isn't aware of. To do this, add a migration without making any model change; an empty migration will be generated, which you can then populate with raw SQL operations.
+
+For example, the following migration creates a SQL Server stored procedure:
+
+```c#
+migrationBuilder.Sql(
+@"
+    CREATE PROCEDURE getFullName
+        @LastName nvarchar(50),
+        @FirstName nvarchar(50)
+    AS
+        RETURN @LastName + @FirstName;
+");
+```
+
+This can be used to manage any aspect of your database, including:
+
+* Stored procedures
+* Full-Text Search
+* Functions
+* Triggers
+* Views
+
+## Remove a migration
+
+Sometimes you add a migration and realize you need to make additional changes to your EF Core model before applying it. To remove the last migration, use this command.
+
+### [.NET Core CLI](#tab/dotnet-core-cli)
+
+```dotnetcli
+dotnet ef migrations remove
+```
+
+### [Visual Studio](#tab/vs)
+
+``` powershell
+Remove-Migration
+```
+
+***
+
+After removing the migration, you can make the additional model changes and add it again.
+
+> [!WARNING]
+> Take care not to remove any migrations which are already applied to production databases.
```

Comment:
...because you won't be able to revert it and it may break the assumptions made by subsequent migrations